samples/amber.cc

```diff
@@ -498,7 +505,10 @@ int main(int argc, const char** argv) {
         std::cerr << options.buffer_filename << std::endl;
       } else {
         for (const amber::BufferInfo& buffer_info : amber_options.extractions) {
-          if (buffer_info.buffer_name == options.fb_name)
+          // Skip frame buffers.
+          if (std::any_of(
+                  options.fb_names.begin(), options.fb_names.end(),
+                  [&](std::string s) { return s == buffer_info.buffer_name; }))
```

Comment:
This should also make sure to check the default generated framebuffer name as it won't be in the fb_names if there were no image filenames specified.